modules/strelka/1.1/strelka.smk

```diff
@@ -233,7 +273,7 @@ def _strelka_get_output(wildcards):
         vcf_files = "variants"
     else:
         vcf_files = ["somatic.snvs", "somatic.indels"]
```

Comment:
I've tried various iterations of this. It works this way as expected. It doesn't work the other way because then Snakemake never knows about the two separate files made in the paired mode. Those are specified here to force both to be run through earlier rules